tests/Util/YamlSourceManipulatorTest.php

```diff
@@ -58,7 +58,8 @@ private function getYamlDataTests()
         foreach ($finder as $file) {
             list($source, $changeCode, $expected) = explode('===', $file->getContents());
 
-            $data = Yaml::parse($source);
+            // Multiline string ends with an \n
+            $data = Yaml::parse(trim($source, "\n"));
```

Comment:
We actually trim this below already. So it looks like we just need to trim it earlier. I'm thinking the whole inside of the `foreach` would be this:

```php
list($source, $changeCode, $expected) = explode('===', $file->getContents());

$source = rtrim($source, "\n");
$expected = ltrim($expected, "\n");

$data = Yaml::parse($source);
eval($changeCode);

yield $file->getFilename() => [
    'source' => $source,
    'newData' => $data,
    'expectedSource' => $expected,
];
```

Reply:
Good catch, I missed this !